angular/projects/admin/src/app/admin/content-type/content-type-form/content-type-form.component.scss

```diff
@@ -36,3 +36,11 @@
     margin-left: 32px;
   }
 }
+
+.icon-sub-toolbar {
+  padding-top: 4px;
+}
+
+.sub-toolbar-spacer {
+  flex: 1 1 auto;
+}
```

Comment:
It seems wrong that we're repeating this in every component that is using the sub-toolbar.
Do like this instead: the sub toolbar has the padding defined in the navigation component and then it has 2 parts
 1. Breadcrumbs
 2. Action items/buttons

Then you define the flex-spacing between the "breadcrumbs" and "actions".

Reply:
Something like this would be more generic and not so "aware" of that it is going to be rendered as a toolbar.

```html
<bread-crumb>
    Content Type <mat-icon class="icon-sub-toolbar">keyboard_arrow_right</mat-icon> {{documentTypeForm.controls['title'].value}}
</bread-crumb>
<actions>
    <button mat-button
            class="sub-toolbar-icon"
            color="warn"
            (click)="cancelEditing()">
      <mat-icon>keyboard_backspace</mat-icon> Cancel
    </button>
    <button mat-button
            class="sub-toolbar-icon"
            (click)="editTemplate()">
      <mat-icon>mode_edit</mat-icon> Edit Template
    </button>
    <button mat-button
            class="sub-toolbar-icon"
            color="primary"
            (click)="save()">
      <mat-icon>save</mat-icon> Save
    </button>
    <button mat-button
            class="sub-toolbar-icon"
            color="warn">
      <mat-icon>delete</mat-icon> Delete
    </button>
</actions>
```